src/app/components/ReadTime/index.tsx

```diff
@@ -145,28 +149,36 @@ export const ReadTime = ({
 
   if (!validRender) return null;
 
-  const { readTimeInMilliseconds, minutesLabel, copy } = ProcessReadTime({
+  const { readTimeInMilliseconds, copy } = ProcessReadTime({
     readTimeValue: readTimeValue as number,
     readTimeVariant: readTimeVariant as string,
   });
 
-  const eventTrackingData: EventTrackingData = {
+  const optimizelyTrackingData: EventTrackingData = {
     componentName: 'read-time',
     sendOptimizelyEvents: true,
     experimentName: 'newswb_ws_homepage_read_time',
     experimentVariant: readTimeVariant,
+  };
+
+  const eventTrackingData: EventTrackingData = {
+    ...optimizelyTrackingData,
     itemTracker: {
-      label: `Read time: ${readTimeValue} ${minutesLabel}`,
+      type: promoType,
+      position: promoPosition,
+      label: `Read time: ${readTimeValue} ${readTimeValue === 1 ? 'minute' : 'minutes'}`,
```

Comment:
Updated this since it seems to make more sense just to keep this untranslated for our own viewing.